lib/modules/github.js

```diff
@@ -79,6 +79,9 @@ exports.setup = function(everyauth) {
 
     return true;
   };
+  everyauth.github.username = function(req, auth) {
```

Comment:
Should this function be `everyauth.github.decorate`, to match the others? Also, are there scenarios where we have an `auth` object but don't have the full `auth.*.user.email`? I'm just wondering if we need to guard against those cases.